cam0:
  intrinsics:
    - 700.6512811598237
    - 700.4762153007572
    - 756.5124331276594
    - 556.1420040326701
  resolution:
    - 1440
    - 1080
  distortion_model: equidistant
  T_bundle_camera:
    - - 1
      - 0
      - 0
      - 0
    - - 0
      - 1
      - 0
      - 1.214306433183765e-17
    - - 0
      - 0
      - 1
      - -5.551115123125783e-17
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/alphasense_driver_node/cam1
  distortion_coeffs:
    - -0.03739677968839148
    - -0.0067434198368627
    - 0.006337002943966249
    - -0.002658471748273983
cam1:
  intrinsics:
    - 702.4755857158741
    - 701.6270253125284
    - 763.1743144498663
    - 544.3852457555274
  resolution:
    - 1440
    - 1080
  distortion_model: equidistant
  T_bundle_camera:
    - - 0.9999935136192531
      - 0.003328648282317174
      - -0.001375797962523997
      - -0.0551802904964161
    - - -0.003326828952264415
      - 0.9999935914994983
      - 0.001322561551477531
      - -0.00128085653103486
    - - 0.001380191487958648
      - -0.001317975928345556
      - 0.9999981790037967
      - -0.0005097051019628992
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/alphasense_driver_node/cam2
  distortion_coeffs:
    - -0.04510436708921242
    - 0.01475074178331407
    - -0.01535909618142446
    - 0.004554297236190726
cam2:
  intrinsics:
    - 702.2304652317938
    - 700.8418016215869
    - 758.2752158071359
    - 541.4257848035661
  resolution:
    - 1440
    - 1080
  distortion_model: equidistant
  T_bundle_camera:
    - - 0.9998537844880657
      - -0.01708577159655839
      - 0.0006971756184744703
      - 0.05530576377218068
    - - 0.0170852504107707
      - 0.9998537575619653
      - 0.0007467982460049316
      - 0.0003129050497772663
    - - -0.0007098332860722367
      - -0.00073477763249516
      - 0.9999994781191321
      - 2.8029725606929e-05
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/alphasense_driver_node/cam3
  distortion_coeffs:
    - -0.03742776055211713
    - -0.004921684280563942
    - 0.004815064416594661
    - -0.002217640353731962
cam3:
  intrinsics:
    - 699.9009769846375
    - 699.97830848032
    - 707.4339968730018
    - 524.6573172726385
  resolution:
    - 1440
    - 1080
  distortion_model: equidistant
  T_bundle_camera:
    - - 0.004491486004606805
      - -0.1634515954637117
      - 0.986541130665031
      - 0.1027760110132825
    - - 0.1695000497778596
      - -0.9721511344536004
      - -0.1618391328013133
      - -0.01185405817658139
    - - 0.9855199438259855
      - 0.1679456689556034
      - 0.02333865035260874
      - -0.06795597193196336
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/alphasense_driver_node/cam4
  distortion_coeffs:
    - -0.03570412886568466
    - -0.01146195954145682
    - 0.01052423965798831
    - -0.003758048076511013
cam4:
  intrinsics:
    - 698.2127010470355
    - 698.0112950267592
    - 690.8838700455492
    - 526.8941342772863
  resolution:
    - 1440
    - 1080
  distortion_model: equidistant
  T_bundle_camera:
    - - -0.004569398394279263
      - -0.1889392662378434
      - -0.9819781434796906
      - -0.1029472033280969
    - - 0.1600244405376981
      - 0.9691951381655061
      - -0.1872243642984064
      - -0.0138030902450006
    - - 0.9871024764576934
      - -0.1579960057401801
      - 0.02580626166723731
      - -0.06773204451441744
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/alphasense_driver_node/cam5
  distortion_coeffs:
    - -0.03946270390313153
    - 0.0006176457946188308
    - -0.001470857524915949
    - 0.0001217080209412438
cam5:
  intrinsics:
    - 985.5074474241898
    - 985.0423099311803
    - 951.7960840489374
    - 638.127222917076
  resolution:
    - 1920
    - 1280
  distortion_model: equidistant
  T_bundle_camera:
    - - -0.9999762044278094
      - 0.00344876264213937
      - -0.005974664374681691
      - -0.0004317665062718096
    - - -0.004429376889316839
      - -0.9849469356358074
      - 0.1728002158621487
      - -0.03406260620994009
    - - -0.005288780438276161
      - 0.1728225680224403
      - 0.9849387741294403
      - 0.006600347759186874
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/hdr_right/image_raw
  distortion_coeffs:
    - -0.06129863822544315
    - 0.002526270977915111
    - -0.0004191605011825837
    - -0.0001852354097388015
cam6:
  intrinsics:
    - 988.9032668287883
    - 988.8389268301031
    - 943.7610293419009
    - 637.9507018827538
  resolution:
    - 1920
    - 1280
  distortion_model: equidistant
  T_bundle_camera:
    - - 0.1731355067563372
      - 0.01823193069161482
      - 0.9847292485772332
      - 0.1039628418856186
    - - 0.1748233955086735
      - -0.9845203369186988
      - -0.01250945947311795
      - -0.046330113810259
    - - 0.9692578999848513
      - 0.1743195424981026
      - -0.1736427954745257
      - -0.06778651498810374
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/hdr_left/image_raw
  distortion_coeffs:
    - -0.06032840433344838
    - -0.0002366168999947844
    - 0.001647358267562969
    - -0.0008027475861239494
cam7:
  intrinsics:
    - 984.8090864251797
    - 984.5533102739237
    - 948.0326079931228
    - 636.7121181864717
  resolution:
    - 1920
    - 1280
  distortion_model: equidistant
  T_bundle_camera:
    - - 0.1747255378077741
      - 0.007246750042052165
      - -0.984590509324365
      - -0.1036266390168784
    - - -0.1652438995826785
      - -0.9855741928656909
      - -0.03657819033046113
      - -0.0479709934765629
    - - -0.970652069532896
      - 0.1690887192303786
      - -0.171007499661671
      - -0.06798262742658699
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/hdr_front/image_raw
  distortion_coeffs:
    - -0.05944588809639628
    - 0.002099898204538727
    - -0.001404527973232062
    - 0.000335312479820808
cam8:
  intrinsics:
    - 1061.718625035199
    - 1062.395168315549
    - 953.9596793910416
    - 554.2653558995383
  resolution:
    - 1920
    - 1280
  distortion_model: radtan
  T_bundle_camera:
    - - 0.9999215681147087
      - -0.01217222126324004
      - -0.002948668943933777
      - -0.05860914968743371
    - - 0.01217580369146871
      - 0.9999251520522738
      - 0.001200040709321574
      - -0.07494782193721314
    - - 0.002933841081076042
      - -0.001235849002078783
      - 0.9999949326140383
      - 0.02058089561875001
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/zed2i_driver_node/left_raw/image_raw_color
  distortion_coeffs:
    - -0.05188431064767035
    - 0.01844333610963792
    - 0.0005809017649331493
    - -9.738012245307233e-05
cam9:
  intrinsics:
    - 1061.564494160665
    - 1062.514514351718
    - 954.7187652702173
    - 551.8916947859017
  resolution:
    - 1920
    - 1280
  distortion_model: radtan
  T_bundle_camera:
    - - 0.9999419272590334
      - -0.009744729975530985
      - -0.004602428401830738
      - 0.06127172817250662
    - - 0.009743110990388019
      - 0.9999524648045018
      - -0.0003740583250725235
      - -0.07388537874699955
    - - 0.00460585472186982
      - 0.0003291946317359541
      - 0.9999893388097573
      - 0.0215899120395959
    - - 0
      - 0
      - 0
      - 1
  rostopic: /gt_box/zed2i_driver_node/right_raw/image_raw_color
  distortion_coeffs:
    - -0.05270490201887633
    - 0.01958353210072089
    - 1.232285966101087e-05
    - -0.0001750751863060589